### DISCUSSION TITLE
Give us a way to pay you!

### DISCUSSION THREAD

**Discussion by dana-ross** [kazad/instacalc#90] [Ideas]
Instacalc is exactly what I was looking for as a sort of "spreadsheet scratchpad". I'd love a way to donate to support its development or at least buy you a glass of your favorite beverage!

**Comment by kazad** [comment]
Glad you're enjoying it Dana! I'll add a feature to help support the site :).

**Comment by kazad** [comment]
Just added donation links to the about section:

Paypal: https://www.paypal.com/donate/?hosted_button_id=7H4VQ8MCEWK5U
Github: https://github.com/sponsors/kazad

Thanks for the suggestion!
